README.md

```diff
@@ -41,39 +41,49 @@ This is the dependency you need:
 </dependency>
 ```
  
-Then, you implement `com.yegor256.rpm.Storage` interface.
+Then, use one of the existing `com.artipie.asto.Storage` implementations to create the `Storage`,
+[`FileStorage`](https://github.com/artipie/asto/blob/master/src/main/java/com/artipie/asto/fs/FileStorage.java), 
+[`S3`](https://github.com/artipie/asto/blob/master/src/main/java/com/artipie/asto/s3/S3Storage.java) 
+and other storages are supported. Or you can implement `com.artipie.asto.Storage` by yourself.
 
 Then, you make an instance of `Rpm` class with your storage
-as an argument. Finally, you put your artifacts to the storage
-and instruct `Rpm` to update the meta info:
+as an argument. Finally, you put your artifacts to the storage specifying repository key 
+(`rpm-repo` in our example) and instruct `Rpm` to update the meta info:
 
 ```java
 import com.artipie.rpm.Rpm;
-Rpm rpm = new Rpm(storage);
-rpm.update("test/my.rpm");
+final Rpm rpm = new Rpm(storage);
```

Comment:
@olenagerasimova maybe we can expand this example with instantiation of `FileStorage` jsut to make it more obvious.